msteams-platform/bots/what-are-bots.md

```diff
@@ -57,6 +57,8 @@ You can change the bot name displayed in Teams environment, and you need to upda
 
   :::image type="content" source="~\assets\images\bot_name.png" alt-text="The screenshot shows the bot name displayed in a Teams window.":::
 
+[!INCLUDE [requirements-targeting](../includes/requirements-targeting.md)]
```

Comment:
Since this article is the general entry point for bot scenarios (standalone *and also* bot apps + other capabilities), I'd argue that the RT include should remain. We'd want to alert customers about this option, and they can decide if its relevant to their scenario.

Reply:
Fair point, @erikadoyle.